[Task-975]-Need to remove profile page and my account page

rajiravenantech opened the pull request:
Need to remove profile page and my account page

rajiravenantech committed:
[Task-975]-Removed the profile and my accocunt page , Notification list.

kulanthairavenantech reviewed (changes requested):
- Remove notification icon ind inbox icon
- Add the logout link in the drop-down

rajiravenantech committed:
[Task-975]-Removed notification and mail icon in header.

rajiravenantech commented:
> * Remove notification icon ind inbox icon
> * Add the logout link in the drop-down

Removed notification icon ind inbox icon. Added logout menu in header.
